workflow/Snakefile

```diff
@@ -193,22 +194,22 @@ rule FILTER:
         queue=search("queue", "FILTER"),
         walltime=search("walltime", "FILTER"),
     run:
-        # Filter out variants with >= 20% missingness
+        # Filter out variants with >= 100% missingness
         shell(
-            "module load plink-2; plink2 --allow-extra-chr --vcf {input} --geno 0.2 --output-chr chr26 --export vcf-4.2 bgz --out results/FILTER/ALL_FILTERED1"
+            "module load plink-2; plink2 --allow-extra-chr --vcf {input} --geno 1 --output-chr chr26 --export vcf-4.2 bgz --out results/FILTER/ALL_FILTERED1"
```

Comment:
Is a decimal point necessary after the "1"?

Reply:
I have downloaded Plink-2.0 on my local Windows and tested. No decimal is required for 100% missingness tests thankfully.